util/libssl.num

```diff
@@ -520,3 +520,4 @@ SSL_load_client_CA_file_ex              520	3_0_0	EXIST::FUNCTION:
 SSL_set0_tmp_dh_pkey                    521	3_0_0	EXIST::FUNCTION:
 SSL_CTX_set0_tmp_dh_pkey                522	3_0_0	EXIST::FUNCTION:
 SSL_group_to_name                       523	3_0_0	EXIST::FUNCTION:
+SSL_client_hello_get_raw_extensions     524	3_0_0	EXIST::FUNCTION:
```

Comment:
This should be an unnumbered entry for now i.e. `?` instead of 524. Also the version number must be `3_1_0`.

Reply:
done.